docs/cli.md

```diff
@@ -30,50 +30,118 @@ $ sematic run examples/mnist/pytorch
 
 SNOWFLAKE_USER: foobar
 ```
 
-These settings are simply store in the `/.sematic/settings.yaml` file on
-your machine.
+These settings are simply stored in the `~/.sematic/settings.yaml` file on
+your machine (the "default" qualifier is a placeholder for a future feature):
+
+```shell
+$ cat ~/.sematic/settings.yaml
+default:
+  SNOWFLAKE_USER: foobar
+```
 
 You can always override them at runtime with an environment variable:
 
 ```shell
 $ SNOWFLAKE_USER="notfoobar" sematic settings show
-Active settings:
+Active user settings:
 
 SNOWFLAKE_USER: notfoobar
 ```
 
 If you'd like, you can even change which directory Sematic uses to hold its
 settings by setting the environment variable `SEMATIC_CONFIG_DIR`. This can
-be either a relative path (it will be treated as relative to your home directory)
-or an absolute path.
+be either a relative path (it will be treated as relative to your home
+directory) or an absolute path.
+
+This is the full list of supported user settings:
+- `SEMATIC_API_ADDRESS`: the address of the Sematic server to use for pipeline
+  executions
+- `SEMATIC_API_KEY`: the user's personal API key, used to associate pipeline
+  submissions
+- `SNOWFLAKE_USER`: the Snowflake user
+- `SNOWFLAKE_PASSWORD`: the Snowflake password
+- `SNOWFLAKE_ACCOUNT`: the Snowflake account
+
+### Server settings
+
+The same way that certain settings need to be kept on the user side, certain
+ones need to configure the backend Server.
+
+These can be accessed and modified just like the user settings, but by using
+the `sematic server-settings` command instead:
+
+```shell
+$ sematic server-settings show
+Active server settings:
+
+AWS_S3_BUCKET: XXX
+KUBERNETES_NAMESPACE: default
+SEMATIC_AUTHORIZED_EMAIL_DOMAIN: XXX
+SEMATIC_AUTHENTICATE: 'true'
+GOOGLE_OAUTH_CLIENT_ID: XXX
+GRAFANA_PANEL_URL: XXX
+
+```
+
+These settings are simply stored in the `~/.sematic/server.yaml` file on the
+machine which needs to start a backend Server (this file does not have the
+settings nested under a "default" entry):
+
+```shell
+$ cat ~/.sematic/server.yaml
+AWS_S3_BUCKET: XXX
+KUBERNETES_NAMESPACE: default
+SEMATIC_AUTHORIZED_EMAIL_DOMAIN: XXX
+SEMATIC_AUTHENTICATE: 'true'
+GOOGLE_OAUTH_CLIENT_ID: XXX
+GRAFANA_PANEL_URL: XXX
+```
+
+They can also be overridden via environment variables, just like the user
+settings.
+
+This is the full list of supported server settings:
+- `SEMATIC_AUTHENTICATE`: whether to require authentication on API calls
+- `SEMATIC_AUTHORIZED_EMAIL_DOMAIN`: the email domain that is allowed to create
+  accounts in the UI
+- `SEMATIC_WORKER_API_ADDRESS`: the address of the remote cloud worker API,
+  such as the Kubernetes API
+- `GOOGLE_OAUTH_CLIENT_ID`: the Google OAuth client ID to use for
+  authentication
+- `GITHUB_OAUTH_CLIENT_ID`: the GitHub OAuth client ID to use for
+  authentication
+- `KUBERNETES_NAMESPACE`: the namespace to use for Kubernetes jobs
+- `GRAFANA_PANEL_URL`: the URL of the Grafana deployment that tracks jobs
```

Comment:
Nit: I would add that this setting is optional

Reply:
Done.